client/version/parser_test.go

```diff
@@ -17,6 +17,7 @@ func TestFilenameParser(t *testing.T) {
 		{"kcoin-1.0.0-windows-4.0-amd64.zip", "1.0.0", "windows-4.0", "amd64"},
 		{"kcoin-1.0.0-windows-4.0-amd64.exe.zip", "1.0.0", "windows-4.0", "amd64"},
 		{"kcoin-1.0.0-osx-10.6-amd64.zip", "1.0.0", "osx-10.6", "amd64"},
```

Comment:
We don't use this name
Also, shouldn't we add the filenames we use now? For example, the windows release doesn't have the `-4.0` part, and the OSX doesn't have the `-10.6`

Reply:
tests are outdated 
will change